stories/victory-histogram.js

```diff
@@ -0,0 +1,955 @@
+  .add("time scale", () => <VictoryHistogram bins="month" data={timeData} />)
+  .add("time scale with labels", () => (
+    <VictoryHistogram
+      binSpacing={10}
+      data={timeData}
+      bins="year"
+      labels={({ datum }) => `${datum.x0.getFullYear()} - ${datum.x1.getFullYear()}`}
+    />
+  ))
+  .add("horizontal time scale with labels", () => (
+    <VictoryHistogram
+      horizontal
+      binSpacing={10}
+      data={timeData}
+      bins="year"
+      labels={({ datum }) => `${datum.x0.getFullYear()} - ${datum.x1.getFullYear()}`}
+    />
+  ));
+
+storiesOf("VictoryHistogram.scale", module)
+  .addDecorator(getChartDecorator({ scale: { y: "log" }, domainPadding: 25 }))
+  .add("log scale", () => <VictoryHistogram data={data} />)
+  .add("horizontal log scale", () => <VictoryHistogram horizontal data={data} />);
+
+storiesOf("VictoryHistogram.with other charts", module)
+  .addDecorator(getChartDecorator({ domainPadding: 25, theme: VictoryTheme.material }))
+  .add("with line", () => [
+    <VictoryHistogram key="histogram" data={data} />,
+    <VictoryLine
+      key="line"
+      data={[
+        { x: 0, y: 5 },
+        { x: 10, y: 5 },
+        { x: 20, y: 2 },
+        { x: 30, y: 9 },
+        { x: 50, y: 2 },
+        { x: 60, y: 4 },
+        { x: 80, y: 12 },
+        { x: 120, y: 8 }
+      ]}
+    />
+  ])
+  .add("with scatter", () => [
+    <VictoryHistogram key="histogram" data={data} />,
+    <VictoryScatter
+      key="scatter"
+      data={[
+        { x: 0, y: 5 },
+        { x: 10, y: 5 },
+        { x: 20, y: 2 },
+        { x: 30, y: 9 },
+        { x: 50, y: 2 },
+        { x: 60, y: 4 },
+        { x: 80, y: 12 },
+        { x: 120, y: 8 }
+      ]}
+    />
+  ])
+  .add("with line horizontal", () => [
+    <VictoryHistogram horizontal key="histogram" data={data} />,
+    <VictoryLine
+      horizontal
+      key="line"
+      data={[
+        { x: 0, y: 5 },
+        { x: 10, y: 5 },
+        { x: 20, y: 2 },
+        { x: 30, y: 9 },
+        { x: 50, y: 2 },
+        { x: 60, y: 4 },
+        { x: 80, y: 12 },
+        { x: 120, y: 8 }
+      ]}
+    />
+  ]);
+
+const stackedData = [
+  ...[50, 30, 100, 32, 50, 10, 49, 78, 20].map((count) => getData(count, count, 100)),
+  [{ x: 1 }, { x: 3 }, { x: 1 }, { x: 2 }]
+];
+
+storiesOf("VictoryHistogram.stacked", module)
+  .addDecorator(getChartDecorator())
+  .add("stacked area with explicit bins passed to stack", () => (
+    <VictoryStack colorScale="qualitative" bins={[0, 20, 100]}>
+      {stackedData.map((d, index) => (
+        <VictoryHistogram data={d} key={index} />
+      ))}
+    </VictoryStack>
+  ))
+  .add("stacked area with numeric bins passed to stack", () => (
+    <VictoryStack colorScale="qualitative" bins={2}>
+      {stackedData.map((d, index) => (
+        <VictoryHistogram data={d} key={index} />
+      ))}
+    </VictoryStack>
+  ))
+  .add("stacked area with bins passed to first child", () => (
```

Comment:
based in the title of this story, I would expect a `bins` prop on the first histogram child